tests/test_lab_serve.py

```diff
@@ -18,9 +18,8 @@
 
 
 def assert_vllm_args(args):
-    assert "python" in args[0]
-    assert args[1] == "-m"
-    assert args[2] == "vllm.entrypoints.openai.api_server"
+    assert "vllm" in args[0]
```

Comment:
why use `in` here and not `==`, and vice versa for the assert below? `args[0]` should always be `vllm` and `args[1]` should always be `serve`, right?

Reply:
I'm sure I did this because the old line was `"python" in args[0]`. You're right that `==` would be a little better.